right/src/keyboard_layout.h

```diff
@@ -33,6 +33,32 @@ typedef enum {
     UHK_KEY_LPRESSLAYER,
 } uhk_key_type_t;
 
+enum {
```

Comment:
Looks like you're a living proof that there's someone who uses directional movement keys, which is something I wouldn't have thought. :) Let's implement directional movement and scrolling then, both in the firmware and Agent.

I don't feel strongly regarding bitfields for buttons. I leave it up to you. I'm only concerned about the binary serialization format which should not be changed, but it doesn't affect our in-memory states.